client/src/components/web-extension/Common/Header/OuterHeader.jsx

```diff
@@ -11,7 +11,7 @@ export const OuterHeader = ({ setHeader, header }) => {
 	const currentStep = useSelector(selectCreateWalletCurrentStep);
 	const navigate = useNavigate();
 	const { pathname, state } = useLocation();
-	const shouldShowBackArrow = pathname !== '/welcomeBack';
+	const shouldShowBackArrow = pathname !== '/welcomeBack' && (pathname === '/createWallet' && currentStep !== 0);
```

Comment:
the back arrow should be configurable in route config or some where else.
We can refactor later.